ui/webui/src/components/Error.jsx

```diff
@@ -30,7 +30,34 @@ import { exitGui } from "../helpers/exit.js";
 
 const _ = cockpit.gettext;
 
-export const CriticalError = ({ exception, isBootIso }) => {
+export const bugzillaPrefiledReportURL = (productQueryData) => {
+    const baseURL = "https://bugzilla.redhat.com";
+    const queryData = {
+        ...productQueryData,
+        component: "anaconda",
+    };
+
+    const reportURL = new URL(baseURL);
+    reportURL.pathname = "enter_bug.cgi";
+    Object.keys(queryData).map(query => reportURL.searchParams.append(query, queryData[query]));
+    return reportURL.href;
+};
+
+const addExceptionDataToReportURL = (url, exception) => {
+    const newUrl = new URL(url);
```

Comment:
I guess we don't check for maximum URL length ? Could be something for a followup PR, possibly with corresponding policy for what to cut if we reach the maximum length.

Reply:
Created a Jira issue for that BTW. :)